internal/config/profile.go

```diff
@@ -641,31 +641,42 @@ func (p *Profile) Save() error {
 	return viper.WriteConfigAs(p.Filename())
 }
 
-// MongoCLIConfigHome retrieves configHome path based used by mongoCLI.
-func MongoCLIConfigHome() (string, error) {
+// OldMongoCLIConfigHome retrieves configHome path based used by mongoCLI.
+// Deprecated: MongoCLI versions below v1.24.0 use this path.
+func OldMongoCLIConfigHome() (string, error) {
 	if home := os.Getenv("XDG_CONFIG_HOME"); home != "" {
-		return home, nil
+		return "", errors.New("not applicable")
 	}
 
 	home, err := os.UserHomeDir()
-
 	if err != nil {
 		return "", err
 	}
 
 	return fmt.Sprintf("%s/.config", home), nil
 }
 
+// MongoCLIConfigHome retrieves configHome path based used by mongoCLI.
+func MongoCLIConfigHome() (string, error) {
+	home, err := os.UserConfigDir()
+	if err != nil {
+		return "", err
+	}
+
+	return fmt.Sprintf("%s/mongocli", home), nil
+}
+
 // AtlasCLIConfigHome retrieves configHome path based used by atlasCLI.
 func AtlasCLIConfigHome() (string, error) {
-	if home := os.Getenv("XDG_CONFIG_HOME"); home != "" {
-		return home, nil
+	home, err := os.UserConfigDir()
+	if err == nil {
+		return fmt.Sprintf("%s/atlascli", home), nil
 	}
 
-	home, err := os.UserHomeDir()
-
-	if err != nil {
-		return "", err
+	home, err = os.UserHomeDir()
```

Comment:
Looks like in `MongoCLIConfigHome` there is no fallback but in `AtlasCLIConfigHome` we do have a fallback to `UserHomeDir`, any reason for the difference?

I'd be inclined to keep things in sync.

Reply:
Without a fallback

Reply:
no reason other than me me forgetting to remove the fallback, synced w/ you offline that is OK to remove for AtlasCLIConfigHome